Gaijinizer patch/Map217.txt

```diff
@@ -1440,549 +1436,711 @@ I am...!?#GT
 
 直前まで安定していて、魔力の暴走も小規模だった
  アンタほどうまくはいかなかったんだよ
-> CONTEXT: Map217/events/5/pages/2/713/Dialogue/hyakki < UNTRANSLATED
+> CONTEXT: Map217/events/5/pages/2/713/Dialogue/hyakki
+But there were a lot of victims left behind in
+exchange for that.
 
+
+Though she was stable right until the last
+minute, and the scale of her rampage was small,
+things didn't work out as well as they did for
+you.#RonIffy
 > END STRING
 
 > BEGIN STRING
 世界の一部は焦土と化し、
 当然、彼女を助けようとした仲間たちも……
 
 魔力を使い果たし、『個』が消滅してしまった。
-> CONTEXT: Map217/events/5/pages/2/718/Dialogue/hyakki < UNTRANSLATED
-
+> CONTEXT: Map217/events/5/pages/2/718/Dialogue/hyakki
+Part of the world was scorched, and her friends
+used up all their magic trying to save her and
+lost their "selves".#Ron
 > END STRING
 
 > BEGIN STRING
 守っていたはずの神社や街が消滅し、大事な仲間さえ失った…
 
  哀しみに押し潰されそうになった彼女は……
 『ある決断』をしたのサ。
-> CONTEXT: Map217/events/5/pages/2/723/Dialogue/hyakki < UNTRANSLATED
-
+> CONTEXT: Map217/events/5/pages/2/723/Dialogue/hyakki
+The shrine and town she was supposed to protect
+were obliterated, she'd even lost her precious
+friends. And so, crushed by sorrow, she decided
+to do "that".#Ron
 > END STRING
 
 > BEGIN STRING
  『ある決断』……？
 
 いったいどんな……
-> CONTEXT: Map217/events/5/pages/2/728/Dialogue/kazuya < UNTRANSLATED
+> CONTEXT: Map217/events/5/pages/2/728/Dialogue/kazuya
+"That"?
 
+What on earth do you mean?#Ron
 > END STRING
 
 > BEGIN STRING
 『捨てた』んだよ。
 
 これまでの、無邪気で幸せだった自分の『個』を……
 あの何もない、虚数空間の彼方に……
-> CONTEXT: Map217/events/5/pages/2/733/Dialogue/hyakki < UNTRANSLATED
-
+> CONTEXT: Map217/events/5/pages/2/733/Dialogue/hyakki
+She discarded her carefree, happy self and tossed
+it into that space of nothingness.#Ron
 > END STRING
 
 > BEGIN STRING
 彼女は、人間を好きだった感情や、心を捨てて
 
 人間界を悪魔から守る最高責任職『長（おさ）』として
 真剣に自分の世界を守る道を選んだのさ。
-> CONTEXT: Map217/events/5/pages/2/738/Dialogue/hyakki < UNTRANSLATED
-
+> CONTEXT: Map217/events/5/pages/2/738/Dialogue/hyakki
+She discarded her feelings and love for humanity,
+and chose to take up supreme responsibility for
+earnestly defending the human realm, her world,
+from demons.#RonIffy
 > END STRING
 
 > BEGIN STRING
 だから、アンタが出会った『夜深』は、
 『本物』には違いないけど、正確には『本人』じゃない。
 
  虚数空間を漂う、過去の彼女の『個』なんだよ。
-> CONTEXT: Map217/events/5/pages/2/743/Dialogue/hyakki < UNTRANSLATED
-
+> CONTEXT: Map217/events/5/pages/2/743/Dialogue/hyakki
+So while the Yami you met in there is definitely
+the real her, it's not exactly the "true" her,
+but her "self" from the past floating in empty,
+imaginary space.#Ron
 > END STRING
 
 > BEGIN STRING
 もともと、仲間を大事にする寂しがりやだったからね…
 
 アンタが想ひ魂で虚数空間に干渉しているのを感じて
 構って欲しくて接触してきたんだろうサ。
-> CONTEXT: Map217/events/5/pages/2/748/Dialogue/hyakki < UNTRANSLATED
+> CONTEXT: Map217/events/5/pages/2/748/Dialogue/hyakki
+You see, she was always someone who got lonely
+easily and really loved her friends.
+
 
+So when you turned up and interfered in that
+imaginary space, she felt your presence and
+wanted you to keep her company which is why
+she contacted you.#RonIffyWording
 > END STRING
```

Comment:
Rewording this and swapping around the ordering to make this flow better... (let me know if this is in line with the JP)
```
When you turned up and interacted with that
imaginary space, she felt your presence.

 
You see, she was always someone who got lonely
easily and really loved her friends. She wanted
you to keep her company, which is why
she approached you.
```